net/kadnode/Makefile

```diff
@@ -2,14 +2,14 @@ include $(TOPDIR)/rules.mk
 
 
 PKG_NAME:=kadnode
-PKG_VERSION:=2.2.3
-PKG_RELEASE:=1
+PKG_VERSION:=2.2.5
+PKG_RELEASE:=0
```

Comment:
just leave it at 1.  you only need to change this from 1 if you make packaging changes without changing the version.

Reply:
Isn't it set back to 0 on every release?

Reply:
You'd be the first... :)

```
$ grep -r PKG_RELEASE * | cut -d: -f2,3 | sort | uniq -c | sort -gr | head
    557 PKG_RELEASE:=1
    142 PKG_RELEASE:=2
     40 PKG_RELEASE:=3
     24 PKG_RELEASE:=4
     15 PKG_RELEASE:=5
     14 PKG_RELEASE:=6
     13 PKG_RELEASE=$(PKG_SOURCE_VERSION)
     13 PKG_RELEASE=1
      5 PKG_RELEASE:=8
```

It goes back to 1.  See also https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain

Reply:
Convinced :-)